### PR TITLE
Fix --rpc_ip and --rpc_port (RIPD-679)

### PR DESCRIPTION
**Pull request by vinniefalco**
This reverts commit 2b040569e79280a2b5071de540601f71bf06fe18.

### CODE REVIEW

**Comment by ximinez** [comment]
Reviewers: @ximinez ???

File: src/ripple/net/impl/RPCCall.cpp [hunk 1]
```diff
@@ -1019,7 +1019,21 @@ int RPCCall::fromCommandLine (const std::vector<std::string>& vCmd)
...
-            auto const setup = setup_ServerHandler(getConfig(), std::cerr);
+            ServerHandler::Setup setup;
+            try
+            {
+                std::stringstream ss;
+                setup = setup_ServerHandler(getConfig(), ss);
+            }
+            catch(...)
+            {
+                // eat this
```
Comment (sublimator) [inline src/ripple/net/impl/RPCCall.cpp]: This will bite

Comment (ximinez) [inline src/ripple/net/impl/RPCCall.cpp]: But it allows the user to run the command line client without a config file.

Comment (vinniefalco) [inline src/ripple/net/impl/RPCCall.cpp]: Exactly

Comment (sublimator) [inline src/ripple/net/impl/RPCCall.cpp]: Couldn't you make it a more specific catch?

Comment (vinniefalco) [inline src/ripple/net/impl/RPCCall.cpp]: In all cases we want the show to go on - what do you suggest?

Comment (sublimator) [inline src/ripple/net/impl/RPCCall.cpp]: Shrug, just I've usually ultimately been bitten when I use a catch all.

I'm coming from a Python perspective though, where exceptions are more
pervasive.

I don't know what exceptions this throws.

Comment (vinniefalco) [inline src/ripple/net/impl/RPCCall.cpp]: Make no mistake, this is a hack to fix rpc-port. We are going to move the command line client out of rippled soon anyway.

Comment (ximinez) [inline src/ripple/net/impl/RPCCall.cpp]: Usually I agree, but this is one of the exceptions.

(See what I did there?)

It's done intentionally, it's noted with a (short) comment, it's very narrow in scope (parsing the config file), and the consequences are minimal (only relevant when using the command-line client).

**Comment by ximinez** [comment]
Looks good. Built and tested the options locally. :+1:

**Comment by scottschurr** [comment]
Looks okay as far as I understand it.  I'm glad that Ed looked it over too.
:+1:

File: src/ripple/net/impl/RPCCall.cpp (hunk 1 above)
Comment (sublimator) [inline src/ripple/net/impl/RPCCall.cpp]: Bah, there's no better way of determining if the config file exists or not?

Comment (vinniefalco) [inline src/ripple/net/impl/RPCCall.cpp]: Figure out where the config file is located is a hard problem, and not one worth solving again for this temporary code that will go away when we move the command line client to another tool.

**Pull request closed without merging**